### PR TITLE
[PM-11267] Extension Refresh Events

### PR DESCRIPTION
**Pull request by nick-livefront**
## 🎟️ Tracking

[PM-11267](https://bitwarden.atlassian.net/browse/PM-11267)

## 📔 Objective

Add events to the extension refresh components to match the current event implementations.

|Event|Location|
|-|-|
|`Cipher_ClientViewed`| View, Edit |
|`Cipher_ClientToggledPasswordVisible`| View, Edit |
|`Cipher_ClientToggledHiddenFieldVisible`| View, Edit |
|`Cipher_ClientToggledCardCodeVisible`| View, Edit |
|`Cipher_ClientToggledCardNumberVisible`| View, Edit|
|`Cipher_ClientToggledTOTPSeedVisible`| Edit |
|`Cipher_ClientCopiedPassword`| Vault, View |
|`Cipher_ClientCopiedHiddenField` | View |
| `Cipher_ClientCopiedCardCode`| Vault, View|

## ⏰ Reminders before review

- Contributor guidelines followed
- All formatters and local linters executed and passed
- Written new unit and / or integration tests where applicable
- Protected functional changes with optionality (feature flags)
- Used internationalization (i18n) for all UI strings
- CI builds passed
- Communicated to DevOps any deployment requirements
- Updated any necessary documentation (Confluence, contributing docs) or informed the documentation team

## 🦮 Reviewer guidelines

- 👍 (`:+1:`) or similar for great changes
- 📝 (`:memo:`) or ℹ️ (`:information_source:`) for notes or general info
- ❓ (`:question:`) for questions
- 🤔 (`:thinking:`) or 💭 (`:thought_balloon:`) for more open inquiry that's not quite a confirmed issue and could potentially benefit from discussion
- 🎨 (`:art:`) for suggestions / improvements
- ❌ (`:x:`) or ⚠️ (`:warning:`) for more significant problems or concerns needing attention
- 🌱 (`:seedling:`) or ♻️ (`:recycle:`) for future improvements or indications of technical debt
- ⛏ (`:pick:`) for minor or nitpick changes

[PM-11267]: https://bitwarden.atlassian.net/browse/PM-11267?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: libs/vault/src/cipher-view/custom-fields/custom-fields-v2.component.ts
Comment (shane-melton) [inline libs/vault/src/cipher-view/custom-fields/custom-fields-v2.component.ts:1]: ⛏️ (non-blocking): Small nit, but I think we are able to  remove the separate `fields` input and rely solely on `cipher` like the other view components. (similar to what you did for the card details section).

Comment (nick-livefront) [reply libs/vault/src/cipher-view/custom-fields/custom-fields-v2.component.ts:1]: Nice catch! [99682ff](https://github.com/bitwarden/clients/pull/10832/commits/99682ff1dd5b6a8f14d110101f7a5bbaa523b20c)

File: libs/vault/src/components/copy-cipher-field.directive.ts
[the remark this answers is not in the document]
Comment (nick-livefront) [reply libs/vault/src/components/copy-cipher-field.directive.ts:1]: I didn't notice that, but I added the organization id of the cipher to the copy call and they're being sent as I'd expect: [52eb68c](https://github.com/bitwarden/clients/pull/10832/commits/52eb68c84deb6ea378f694620577cab2e5b35858)

**Review by shane-melton — DISMISSED** [review]
🚀

**Comment by shane-melton** [comment]
Looks like we might need to update the `cipher-form.stories.ts` as they're not longer rendering as expected 😕

**Comment by nick-livefront** [comment]
> Looks like we might need to update the `cipher-form.stories.ts` as they're not longer rendering as expected 😕

🤔 @shane-melton  Are you manually checking storybook? I should get that habit, or is there an automatic check that is getting flagged which I just miss?

Fixed: [2f4bc89](https://github.com/bitwarden/clients/pull/10832/commits/2f4bc893f0c7447269ca57832aee91f7948eacf4)

**Review by shane-melton — APPROVED** [review]

**Pull request merged**